helm-chart/kuberay-apiserver/templates/service.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     prometheus.io/path: /metrics
     prometheus.io/scrape: "true"
-    prometheus.io/port: "8080"
+    prometheus.io/port: "8888"
```

Comment:
What's the reason behind this change?

Reply:
To keep it alligned with the service definition in `deploy/base/appiserver.yaml` and the code behavior (metrics are exposed on port `8888`).

Here's the relevant bit in the `yaml` above:

```yaml
apiVersion: v1
kind: Service
metadata:
  name: kuberay-apiserver
  annotations:
    prometheus.io/path: /metrics
    prometheus.io/scrape: "true"
    prometheus.io/port: "8888"
spec:

```

Reply:
I have created this #1311 issue. We should probably discuss on that issue what is the best way to integrate.